pkg/cluster-operator/customcluster_helper.go

```diff
@@ -410,7 +410,12 @@ func (r *CustomClusterController) ensureWorkerPodCreated(ctx context.Context, cu
 		return workerPod, nil
 	}
 
-	newWorkerPod := generateClusterManageWorker(customCluster, manageAction, manageCMD, hostName, configName)
+	kubesprayImage, err := getKubesprayImage(kubeVersion)
```

Comment:
dono we need to validate kubernetes version.
Then we will fall into dead loop when this function returns error

Reply:
ok, I added  a log for it.